frontend/src/pages/404.js

```diff
@@ -1,7 +1,21 @@
-import { Typography } from "@mui/material";
+import { Button, Typography } from "@mui/material";
 
 const NotFound = () => {
-  return <Typography variant="h3">404 - Page Not Found</Typography>;
+  return (
+    <div>
+      <Typography variant="h3">Have You Seen this page?</Typography>
+      <img
+        src="https://user-images.githubusercontent.com/90754725/194039761-45ed2b6c-aa69-4070-b1df-6ca2ef90cb48.png"
+        alt=""
+        srcset=""
+      />
+      <Typography variant="body1">
+        Approach with extreme caution! To not attempt to use magic against this
+        site!
+      </Typography>
```

Comment:
Instead of the text and the title about the image, can you maybe integrate it on the image? Maybe just use the image of sirius on the newspaper thing and put the text on it? 
Or maybe something similar?